DEPS

```diff
@@ -179,6 +179,7 @@ hooks = [
   {
     'name': 'download_rust_toolchain_aux',
     'pattern': '.',
+    'condition': 'checkout_linux or checkout_mac or checkout_win or checkout_android',
```

Comment:
this actually doesn't change anything, because ios checkout actually has `checkout_mac` set to True.

Reply:
It definitely fixed the sync failure for ios.